dictcc.el

```diff
@@ -61,6 +61,14 @@
   :type 'string
   :group 'dictcc)
 
+
+(defcustom dictcc-source-languages-alist '("en" "sv" "es" "fr")
+  "List of source languages to select from interactively."
+  :type 'list
+  :group 'dictcc)
```

Comment:
I think it would be a better default to list all languages dict.cc supports. Individual users can then narrow it down further if they want to. Also it would be nice to have the languages as pairs `("en" . "English")` and present them to the user as `English (en)` or so. Though that is probably overkill for now.

Reply:
Low hanging fruit, thanks for that one :-D